frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -15,7 +15,7 @@ import FileSaver from "file-saver";
 
 import scriptsAPI, {
   IScriptBatchSummaryQueryKey,
-  IScriptBatchSummaryResponseV1,
+  IScriptBatchSummaryV1,
```

Comment:
We can retire the old summary endpoint completely and clean up a lot (if not all) of the v1/v2 names by switching over to using the batch status endpoint on this page.  The two keys we access (`created_at` and `script_name`) are present in both APIs so I think it's just a matter of switching which API we call here. I'm ok leaving this for a cleanup task.